packages/snowpack/src/resolve-remote.ts

```diff
@@ -44,7 +42,7 @@ async function resolveDependency(
       );
     }
     if (packageSemver.startsWith('npm:@reactesm') || packageSemver.startsWith('npm:@pika/react')) {
-      logger.fatal(
```

Comment:
I replaced `fatal` with `error` for simplicity as I didn’t think it’d work any differently. But if someone likes this, happy to add it (as an alias for `error` I guess)